.github/workflows/validate-forms-shared-konto.yml

```diff
@@ -28,9 +28,23 @@ jobs:
         run: |
           echo "Pipelines version: 2.3.0"
 
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Cache Docker layers
+        uses: actions/cache@v4
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-docker-${{ hashFiles('**/Dockerfile') }}
+          restore-keys: |
+            ${{ runner.os }}-docker-
+
       - name: Build Docker image on test target
         run: |
-          docker build -t test .
+          docker buildx build --cache-from=type=local,src=/tmp/.buildx-cache --cache-to=type=local,dest=/tmp/.buildx-cache -t test .
```

Comment:
Is there a reason why are we doing this through CLI `run` command? Because as I see it this basically just wraps the docker build command for which we use official [docker build action](https://github.com/docker/build-push-action).

Also, doing docker cache like this is a bit unfortunate as this has a lot of implications. Most important one is a security issue that you are using [public GitHub action storage](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/caching-dependencies-to-speed-up-workflows#about-caching-workflow-dependencies) (warning is what you want to read) to cache layers which are basically plain text files.

A better way would be to use some other [build cache backend](https://docs.docker.com/build/cache/backends/) like registry and point it to our harbor, or if that wouldn't work to use S3, through their official GitHub action.